util/src/main/scala/io/catbird/util/internal/Newtype1.scala

```diff
@@ -0,0 +1,24 @@
+package io.catbird.util.internal
+
+/** INTERNAL API — Newtype encoding for types with one type parameter.
+ *
+ * The `Newtype1` abstract class indirection is needed for Scala 2.10,
```

Comment:
We don't support 2.10, so couldn't these go directly on `FuturePar`?

Reply:
Good point, I'll take a look at it again